test/inbox.ts

```diff
@@ -102,6 +102,29 @@ describe('Inboxes', function () {
 		});
 	});
 
+	it('should list all conversations in the test inbox using query string, finding previously known ones', function () {
+		if (!keys.testInboxId) {
+			throw new Error('Cannot find inbox ID');
+		}
+		return frontInst.inbox.listConversations({
+			inbox_id: keys.testInboxId,
+			page: 1,
+			q: 'q[statuses][]=unassigned&q[statuses][]=assigned'
+		}).then(function (response: InboxConversations) {
+			response._pagination.should.exist;
+			response._pagination.should.include.keys('next', 'prev');
+			response._links.should.exist;
+			response._links.self.should.satisfy(function (message: string) {
+				return _.startsWith(message, `${apiUrl}/inboxes/${keys.testInboxId}/conversations`);
+			});
+			response._links.self.should.satisfy(function (message: string) {
```

Comment:
I think this can be `.should.include('q[statuses][0]=unassigned&q[statuses][1]=assigned')`